rep-0003.rst

```diff
@@ -180,6 +179,48 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2021)
+----------------------------------
+Required Support for:
+
+- Ubuntu Wily (15.10)
+- Ubuntu X (16.04)
+
+Minimum Requirements:
+
+- C++03
+
+  - Some C++11 will be enabled (TODO(esteve): need to determine which ones)
```

Comment:
I would request the following features to help remove some boost dependencies:

* std::mutex, std::lock_guard
* std::shared_ptr, std::weak_ptr
* std::function
* <cstdint> header (used by sdformat)